src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -257,7 +255,7 @@ export class CalciteRadioButton {
   @Listen("click")
   check(event: MouseEvent | FocusEvent): void {
     // Prevent parent label from clicking the first radio when calcite-radio-button is clicked
-    if (this.el.closest("label") && (event.target === this.el || event.target === this.radio)) {
+    if (this.el.closest("label") && event.composedPath().includes(this.el)) {
```

Comment:
This is unrelated. Please separate this out. This should be a `fix`.

Reply:
@jcfranco I had to do this because the test was failing since the radio switched from inline to inline-block.

Reply:
I can skip the test for now if that is what you would like.

Reply:
These two tests are failing from the change from inline to inline-block.

https://github.com/Esri/calcite-components/blob/master/src/components/calcite-label/calcite-label.e2e.ts#L610-L664

Reply:
If this was needed because of the display changes then it's fine. Ignore my comment.